eZ/Bundle/EzPublishCoreBundle/Resources/font/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
Sorry, but we can not bundle that here.
ref: https://www.apache.org/licenses/GPL-compatibility.html

![Miso Solly](https://i.pinimg.com/736x/ee/3e/ed/ee3eed0c5f87baab7a9909f80116f399--funny-memes-quotes.jpg)

Reply:
I've expected that it might be a problem with it. Anyway, I think the safest will be to leave font configuration to the user.